Source/Util/Notice.c

```diff
@@ -55,8 +70,14 @@ void broadcast_server_notice(server_t* server, uint8_t console, const char* mess
     HASH_ITER(hh, server->players, player, tmp)
     {
         if (is_past_join_screen(player)) {
-            if (enet_peer_send(player->peer, 0, packet) == 0) {
-                sent = 1;
+            if(player->client == 'o') {
+                if (enet_peer_send(player->peer, 0, packet_utf8) == 0) {
+                    sent = 1;
+                }
+            } else {
+                if (enet_peer_send(player->peer, 0, packet) == 0) {
+                    sent = 1;
```

Comment:
utf8 packet has to have its own sent flag and delete for it. Otherwise we can cause memory leak.